src/ophys_etl/workflows/db/schemas.py

```diff
@@ -124,6 +124,8 @@ class OphysROI(SQLModel, table=True):
     width: int
     height: int
     is_in_motion_border: bool  # Set at by segmentation
+    empty_roi_mask: bool = False  # Set at trace extraction
```

Comment:
I believe if the roi mask has 0 area then it should be filtered out of the set of rois at the `segment_postprocess` step. This should not be the responsibility of trace extraction and I don't think we need a flag for this. @morriscb what do you think?

Reply:
I think that sounds about right.

Reply:
There's some logic in `segment_postprocess` to do morphological changes to the ROI output by suite2p which is likely how a 0 area roi could form